docs/5g-on-helium/5g-poc.mdx

```diff
@@ -119,20 +117,131 @@ must:
   - Be registered with the SAS (Outdoor Radios Only).
 - Broadcast at least 1 "heartbeat" per Reward Period.
 
-### Genesis Phase 3: Speed Test and Quality of Service
+### Genesis Phase 3: Speed Tests and Quality of Service
 
+> - Reward Period is approximately 24 hours, with Reward submission occurring at approximately 01:30
+>   UTC.
 > - 5G Hotspots perform periodic speed tests which will be considered in reward calculations.
 > - A minimum Download speed, Upload speed, and Latency will be required by each 5G Hotspot to
 >   ensure it is providing quality coverage.
 
-A 5G Hotspot will perform a speed test every 12 hours with the results being averaged over the
-previous 6 tests.
-
 To meet the speed test requirements as outlined in
 [HIP-53](https://github.com/helium/HIP/blob/main/0053-mobile-dao.md#oracle-operations), a 5G Hotspot
 must have a rolling average Download speed of at least 100Mbps **AND** a rolling average Upload
 speed of at least 10Mbps **AND** a rolling average Latency of 50ms or less.
 
+A 5G Hotspot will perform a speed test approximately every 12 hours with the results being averaged
+by the 5G PoC Oracle over the previous `6` tests. This moving average is reset following a speed
+test gap of `>12 hours` and a minimum of `2` speed test results are required to be eligible for
+Genesis Rewards.
+
+:::note Moving Average
+
+The "Moving Average", also known as a "Rolling Average" is the average of the 6 most recent speed
+test results for each download, upload, and latency.
+
+:::
+
+#### Example Calculation Tables
+
+Here are a set of tables to help visualize all of these components that make up the Speed Test
+result.
+
+**Successful Speed Test Series:**
+
+Here all Speed Test Metrics are passing, the 5G Hotspot is providing a high quality connection.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-success.png')} />
+
+**Missing A Single Speed Test:**
+
+This 5G Hotspot has not reported Speed Test data, this will be common for a newly connected 5G
+Hotspot. This 5G Hotspot _will_ be rewarded for all Reward Periods.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-one-miss.png')} />
+
+**Missing Multiple Speed Tests:**
+
+This 5G Hotspot has not reported a series of Speed Test data, this is often a result of a 5g Hotspot
+being disconnected from the internet for a long time. This 5G Hotspot _will not_ earn rewards during
+while Test 5 and 6 are the most recent.
+
+<img className="docsheader" src={useBaseUrl('img/5g-on-helium/speed-test-many-miss.png')} />
+
+**Failing A Speed Test:**
+
+This 5G Hotspot has "failed" its Speed Test on all metrics (orange cells), but the Moving Average
+(light green cells) stays within the passing range. This 5G Hotspot _will_ be rewarded for all
+Reward Periods.
```

Comment:
"for all Reward Periods" is not very clear. Reward Period for which results of the Test 6 are the most recent?